auto_nag/scripts/needinfo_regression_author.py

```diff
@@ -143,6 +159,16 @@ def comment_handler(bug, bug_id):
 
         return bugs
 
+    @staticmethod
+    def _has_editbugs_group(user_info: dict) -> bool:
+        """Check if the user has the editbugs group permissions"""
+
+        # All employees has edit bugs permissions
```

Comment:
Is it actually the case? Can you double check with dkl or glob?

Reply:
Yes, it is. I double-checked with dkl on #bmo: https://mozilla.slack.com/archives/C5F7KFJ4R/p1660769611872729